opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetryBinaryFormatImpl.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.opencensus.internal.Utils;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.propagation.BinaryFormat;
+import io.opentelemetry.api.trace.SpanId;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceId;
+import io.opentelemetry.api.trace.TraceState;
+
+public class OpenTelemetryBinaryFormatImpl extends BinaryFormat {
```

Comment:
```suggestion
class OpenTelemetryBinaryFormatImpl extends BinaryFormat {
```

Reply:
Add a comment something like `OpenCensus's code for BinaryFormat is in the implementation, not API. Since we don't want to depend on the implementation for shim users, we reimplement it here.`

Also, let's just copy https://github.com/census-instrumentation/opencensus-java/blob/0a2f5799a2809b84e4c0da4b2cb743b798ca0fc8/impl_core/src/main/java/io/opencensus/implcore/trace/propagation/BinaryFormatImpl.java 100% - there seems to be no benefit in mapping into the OTel types here